connectivity/drivers/ble/FEATURE_BLE/COMPONENT_CYW43XXX/TARGET_PSOC6/CMakeLists.txt

```
@@ -1,7 +1,9 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-target_sources(mbed-os
-    PRIVATE
+add_library(mbed-os-ble-cyw43xxx INTERFACE)
```

```suggestion
add_library(mbed-os-ble-ps0c6 INTERFACE)
```
What you have has already been created in `connectivity/drivers/ble/FEATURE_BLE/COMPONENT_CYW43XXX/CMakeLists.txt`.

The source below should also be included for `mbed-os-ble-ps0c6`.

---

yes this extends the component. We got two options, either extend the target or add additional and link together.

If we want to add additional, I would name this `mbed-os-ble-cyw43xxx-ps0c6`